tests/testthat/test-report.R

```diff
@@ -0,0 +1,88 @@
+context("report")
+
+report_test <- function() {
+    dat <- data.frame(
+        V1 = c(1, 2, 3),
+        V2 = c("a", "b", "c")
+    )
+
+    report <- data_validation_report()
+
+    validate(dat)  %>%
+        validate_cols(is.numeric)  %>%
+        validate_if(.data$V1 > 0)  %>%
+        add_results(report)
+
+    return(report)
+}
+
+get_advanced_view <- function(log) {
+    lines <- log[grep("type", log):length(log)]
+    lines <- lines[!grepl("^[[:blank:]+-=:_|]*$", lines)]
```

Comment:
That regex definitely would benefit of some comment (with example) of what we are trying to achieve here.

Reply:
thanks @mdubel. comments added in [880768c](https://github.com/Appsilon/data.validator/pull/63/commits/880768c014f4fd31f628bec633164642acec2a68)